### ISSUE TITLE
memory constraints

### ISSUE THREAD

**Issue by klali** [licel/jcardsim#40]
In some cases it might be useful to be able to set memory constraints on transient memory in the simulator.

**Comment by robertbachmann** [comment]
Yes, however you will need to create a sub-class of **TransientMemory**

```java
TransientMemory tm = new YourTransientMemorySubclass();
Simulator simulator = new Simulator(new SimulatorRuntime(tm));
````

**Comment by vladtcvs** [comment]
Absolutely agree about memory constrains, but also it would be great to just have statistics of used EEPROM memory to test for memory leaks and logging access to EEPROM to estimate it's wear
